elastalert/config.py

```diff
@@ -233,13 +234,14 @@ def load_modules(rule, args=None):
     if type(rule['alert']) != list:
         rule['alert'] = [rule['alert']]
     for alert in rule['alert']:
-        if alert in alerts_mapping:
+        if type(alert) == str and alert in alerts_mapping:
```

Comment:
This should be ``if isinstance(alert, basestring) and ...`` so that it could work with unicode.